migrations/1593194295169_add-status-to-action-items.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+    pgm.addColumn('action_items', {
+        status: { type: 'varchar(20)' }
```

Comment:
We should just use `text` as the column type. The only reason to use `varchar(X)` is if we want to force the text to be smaller than the given size

Reply:
is it not more performant or anything? Either way, sounds fine to me

Reply:
google tells me it mainly only matters if you want to put an index on the column. Which we don't. TIL